### DISCUSSION TITLE
Benefits

### DISCUSSION THREAD

**Discussion by Monasalim** [community/community#20502] [GitHub Education]
Thanks for accepting my account.
But how can i get my student offer pack.

**Comment by Tostisto** [comment]
Firt you  need requiere for student  [pack](https://education.github.com/pack). Then you will see your [benefits](https://education.github.com/pack/offers)

**Comment by Monasalim** [comment]
I have been asking for student pack and submitting my application but till now not receive my student offer

احصل على Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Reply by Tostisto** [reply]
Then you can report it [here](https://support.github.com/request/education).

**Comment by Monasalim** [comment]
OK, thanks.

احصل على Outlook for Android<https://aka.ms/AAb9ysg>
________________________________
